### ISSUE TITLE
Path to local CSS files

### ISSUE THREAD

**Issue by plohoj** [ecmel/vscode-html-css#200]
Thanks for the extension, I suggest a couple of improvements for working with local css files:

1. Add the ability to specify several files at once using a mask, for example, as in VSCode:

    ```json
    "css.styleSheets": [
       "/node_modules/some_library/**/*.css",
       "/styles/**/*.css"
    ]
    ```
    Alternatively, you can use a regular expression or the "`.ignore`" style.
    This will help in cases where there are more `CSS` files with global styles.

2. Add the ability to specify files relative to the currently open file, for example, using a variable in the path:
    ```json
    "css.styleSheets": [
       "${fileDirname}/styles/*.css",
       "${fileDirname}/*.component.css"
    ]
    ```
    This will be useful for *Angular* applications in which the `CSS` file is relative to the `HTML` file or for any similar architecture of relative location.

**Comment by DavideViolante** [comment]
I agree, I've been using this extension for long since some days ago that I had to uninstall it becaose it now returns several errors just for local files (Angular project).
The options css.styleSheets should support remote **and** local files, optionally with wildcard support too.

**Comment by ecmel** [comment]
`${fileBasenameNoExtension}` is implemented in https://github.com/ecmel/vscode-html-css/commit/dd1b86d6f3f17135a5532a0f556790bac9f3b028

**Comment by mdsadiqueinam** [comment]
how to properly implement it in django

**Comment by mdsadiqueinam** [comment]
`{
    "css.styleSheets": [
        "https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css",
        "/style.css",
        "style.css",
        "${fileBasenameNoExtension}.css"
    ]
}`
using this only works with bootstrap

**Comment by ecmel** [comment]
Implemented in 1.14.0
